src/views/statutory/EditPositionModal.vue

```diff
@@ -128,6 +140,17 @@ export default {
     this.dates.starts = this.position.starts = new Date(this.position.starts)
     this.dates.ends = this.position.ends = new Date(this.position.ends)
     this.dates.ends_force = this.position.ends_force = new Date(this.position.ends_force)
+
+    this.axios.get(this.services['oms-core-elixir'] + '/bodies').then((response) => {
+      /* The bodies that observe elections during an Agora are:
+       * ALl commissions, SUCT, CD, all WG's
+       */
+      for (const body of response.data.data) {
+        if (body.type == 'commission' || body.type == 'working group' || body.name == 'Summer University Project' || body.name == 'Comité Directeur') {
```

Comment:
I propose to not filter it here and just pass all of them, otherwise if there is some body to be added we'll need another change here